rtb/piano_di_qualifica_ver0.1.0.typ

```diff
@@ -0,0 +1,99 @@
+#import "/templates/template.typ": *
+
+#show: content => verbale(
+  data: "7 Dicembre 2024",
```

Comment:
[] dicembre in minuscolo (vedi NdP sez. 3.1.11)

N.B. ho notato che i documenti generali hanno la maiuscola, è da cambiare

Reply:
Scusa non ho capito cosa intendi con documenti generali.
Intendi tipo Norme di Progetto?

Reply:
Sì nelle NdP ad esempio alla pagina 1 c'è scritto il mese in maiuscolo ma è sbagliato, come nelle PdP, AdR e Glossario, domani avviso tutti